staff/fede-gomez/app-template/login.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Login</title>
+</head>
+<body>
+    <header>
+        <h1>Website</h1>
+    </header>
+    <nav>
+            <a href="./home.html">Home</a>
+            <a href="news.html">News</a>    
+            <a href="profile.html">Profile</a>    
+            <a href="about.html">About</a>    
+            <a href="contact.html">Contact</a>    
+            <a href="register.html">Register</a>    
+            <a href="login.html">Login</a>     
+    </nav>
+    <section>
+        <h1>Login</h1>
+        <form>
+            <p>User</p>
+            <input name="firstname" type="text">
+            <p>Password</p>
+            <input name="password" type="text">
```

Comment:
You could put type = "password". If you do this what the user writes is not visible

Reply:
Indeed. Thanks for the tip :)